cartography/intel/aws/iam.py

```diff
@@ -259,6 +273,27 @@ def load_group_memberships(session, group_memberships, aws_update_tag):
             )
 
 
+def _find_roles_assumable_in_policy(policy_data):
+    ret = []
+    for statement in policy_data["PolicyDocument"]["Statement"]:
+        action = statement.get('Action')
+        if not action:
+            continue
+
+        # TODO actions may contain wildcards, e.g. sts:* -- this can be solved by using policyuniverse to
+        # TODO expand the policy before checking if the sts:AssumeRole action is allowed
+        if action == "sts:AssumeRole":
+            # TODO blanket allows may be modified by subsequent denies... -- does policyuniverse handle?
+            if statement["Effect"] == "Allow":
+                role_arns = statement["Resource"]
+
+                if isinstance(role_arns, str):
+                    role_arns = [role_arns]
+
+                ret.extend(role_arns)
+    return ret
+
+
 def load_group_policies(session, group_policies, aws_update_tag):
     ingest_policies_assume_role = """
     MATCH (group:AWSGroup{name: {GroupName}})
```

Comment:
is that safe in multi AWS setup?

Reply:
As above, I'll address this in a future PR. It's almost certainly not safe in a multi-account situation, we should be using ARNs.

Reply:
since we are touching this now. Would it be better to fix that multi aws account bug that slipped before?